src/request/stores/request-detail-store.js

```diff
@@ -102,3 +105,18 @@ function getTabs(request) {
 
     glimpse.on('data.request.detail.requested', triggerRequest);
 })();
+
+// Toggle Logging Filter Level
+(function () {
+    function triggerRequest(payload) {
+        _viewModel.logging.filter = payload;
+
+        requestChanged(_viewModel);
+    }
+
+    glimpse.on('data.request.detail.logging.filter', triggerRequest);
+})();
```

Comment:
The logging filters are supposed to be sticky, persisting across tabs and selected requests, meaning they outlive the lifetime of the control/model/tab, so this store seems like the right "level" to store that state.  I'm not sure I'd agree that it's "short circuiting" the update chain; the component invokes the component model in response to the user's clicks, the component model transforms the state and publishes the update, the store processes the update and publishes its own update, which is picked up by the detail component, which starts the cascade of re-rendering which will ultimately reflect the filter change.  

(There are lifetime issues with the component model for listening to and relying events because it has no other knowledge about the component lifecycle, for example, with which to properly subscribe/unsubscribe from events and avoid leaks.  I'm a little concerned about introducing those concepts to component models as it feels a little too much like reimplementing parts of Redux, when we might just be better off adopting it itself.)  

`react-redux` has a concept of a "connected component" which is one that has no UI itself but simply manages the properties/states/callbacks of the "presentational" components within it.  Being "connected" to a store, it can dispatch events as well as subscribe to store state (to pick up changes and force re-rendering).  It serves a lot of the same purpose as our component model, but still being a React component means it natively understands the React lifecycle.  We might be able to migrate our idea of a component model to react/redux/react-redux (though the latter requires React 0.14+, to which we have yet to upgrade).